lib/tasks/rss.rake

```diff
@@ -31,9 +31,12 @@ Examples:
 END_DESC
 
   task :create_notifications => :environment do
+    posts = ENV['FOREMAN_RSS_LATEST_POSTS'].to_i
+    posts = 3 if posts == 0
```

Comment:
Wouldn't it be better to set the default only if there is no ENV variable, allowing setting it to 0 to disable?

Reply:
updated